File: templates/RadicalProject VBNet Template/Presentation/MainViewModel.vb
Namespace Presentation
    Public Class MainViewModel
        Inherits Topics.Radical.Windows.Presentation.AbstractViewModel

        Sub New()
            Me.SetInitialPropertyValue(Function() Me.SampleText, "Welcome to Radical.")
        End Sub

        Public Property SampleText As String
            Get
                Return Me.GetPropertyValue(Function() Me.SampleText)
            End Get
            Set(value As String)
                Me.SetPropertyValue(Function() Me.SampleText, value)
            End Set
        End Property

    End Class
End Namespace

File: templates/RadicalProject CSharp Template/Presentation/MainViewModel.cs
using System;
using Topics.Radical.Windows.Presentation;

namespace RadicalTemplate.Presentation
{
	class MainViewModel : AbstractViewModel
	{
		public MainViewModel()
		{
			this.SetInitialPropertyValue( () => this.SampleText, "Welcome to Radical." );
		}

		public String SampleText
		{
			get { return this.GetPropertyValue( () => this.SampleText ); }
			set { this.SetPropertyValue( () => this.SampleText, value ); }
		}
	}
}
